src/js/views/crop_modal_view.js

```diff
@@ -167,6 +179,18 @@ var CropModalView = Backbone.View.extend({
                 if (sameT) {
                     t = self.m.get('theT');
                 }
+
+                var rotation = self.m.get('rotation');
+                if (rotation != 0) {
```

Comment:
could that section of code be turned into a function?
Same code above using ``roi`` instead of ``r``.

Reply:
Fixed in 88cafb4